src/modules/scraper/entry-point/http/controller.ts

```diff
@@ -47,4 +50,18 @@ export class ScraperController {
       });
     }
   }
+
+  @Post("scraper/deposit-filled-date")
+  @ApiTags("scraper")
+  @Roles(Role.Admin)
+  @UseGuards(JwtAuthGuard, RolesGuard)
+  async submitDepositFilledDateJob(@Body() body: SubmitDepositFilledDateBody) {
+    const { fromDepositId, toDepositId } = body;
+
+    for (let depositId = fromDepositId; depositId <= toDepositId; depositId++) {
+      await this.scraperQueuesService.publishMessage<DepositFilledDateQueueMessage>(ScraperQueue.DepositFilledDate, {
+        depositId,
+      });
+    }
+  }
```

Comment:
This endpoint allows to send manually messages to `DepositFilledDate` queue. This way we can compute/re-compute the filled date for any deposit in the DB